cliclient/cliclient_test.go

```diff
@@ -0,0 +1,101 @@
+package cliclient
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"os"
+	"reflect"
+	"runtime"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/rancher/cli/cliclient/mocks"
+	"github.com/rancher/cli/config"
+	"github.com/rancher/norman/types"
+)
+
+func TestCreateClientOpts(t *testing.T) {
+	conf := &config.ServerConfig{
+		URL:       "https://a.b",
+		AccessKey: "AccessKey",
+		SecretKey: "SecretKey",
+		CACerts:   "CACerts",
+	}
+
+	clientOpts := createClientOpts(conf)
+
+	assert.NotNil(t, clientOpts)
+	assert.NotNil(t, clientOpts.HTTPClient)
+	assert.NotNil(t, clientOpts.HTTPClient.Transport)
+	assert.Equal(t, "https://a.b/v3", clientOpts.URL)
+	assert.Equal(t, "AccessKey", clientOpts.AccessKey)
+	assert.Equal(t, "SecretKey", clientOpts.SecretKey)
+	assert.Equal(t, "CACerts", clientOpts.CACerts)
+}
+
+func TestDefaultHTTPClient(t *testing.T) {
+	client := DefaultHTTPClient.New()
+
+	assert.NotNil(t, client)
+	assert.Equal(t, time.Minute, client.Timeout)
+
+	assert.NotNil(t, client.Transport)
+	transport, is := client.Transport.(*http.Transport)
+	assert.True(t, is)
+	assert.NotNil(t, transport)
+	assert.NotNil(t, transport.DialContext)
+	assert.NotNil(t, transport.Proxy)
+	assert.Equal(t, "net/http.ProxyFromEnvironment", runtime.FuncForPC(reflect.ValueOf(transport.Proxy).Pointer()).Name())
+	assert.Equal(t, 10*time.Second, transport.ResponseHeaderTimeout)
+}
+
+func TestMasterClientNewManagementClient(t *testing.T) {
```

Comment:
Can you move it to `TestNewManagementClient` in order to follow the tests naming convention?

Reply:
Done